docs/CHANGELOG.asciidoc

```diff
@@ -63,9 +63,9 @@
 
 === Bug Fixes
 
-* Fixed background persistence of categorizer state (See {ml-pull}1137[#1137],
+* Fixed background persistence of categorizer state. (See {ml-pull}1137[#1137],
   issue: {ml-issue}1136[#1136].)
-
+* Fixed macOS sandbox rules. (See {ml-pull}1146[#1146].)
```

Comment:
Nit, a little more detail can't hurt

Update macOS sandbox rules to allow calling `nice` . (See {ml-pull}1146[#1146].)

Reply:
I was just about to say that the other change, to allow the signal used to cancel blocking IO calls, was more important.  If that is disallowed then it could cause the controller to hang indefinitely if the JVM doesn't connect to it (e.g. when ML is disabled) and then the JVM crashes rather than exiting gracefully.  However, then I realised that the controller doesn't install the sandbox at all, so doesn't suffer that problem.

Given that, I don't think this needs backporting to 7.7 after all and doesn't need separately release noting because it should have just been part of the CPU priority reduction change that's already release noted for 7.8.

So I think I'll mark this a `>non-issue` and remove the release note altogether.